specification/1.0/README.md

```diff
@@ -163,14 +159,11 @@ Examples:
 }
 ```
 
-
 ## Accessors
```

Comment:
It might be better to reverse the order of the chapters here.
From Accessors > Bufferviews > Buffers to Buffers > Bufferviews > Accessors.

Reply:
I fully agree that from buffer to bufferview to accessor is easier to understand than the other way around. In case we do this change, the jump from data items to buffers might appear harsh, and should be explained as well.